### PR TITLE
Make fixity dashboard viewable only by admins

### LINKED ISSUE #899: Access controls for fixity dashboard
should be admin-only

### PR DESCRIPTION
**Pull request by hackmastera**
fixes #899

### CODE REVIEW

File: app/views/catalog/_fixity_dashboard_home.html.erb [hunk 1]
```diff
@@ -1,12 +1,14 @@
-<div class="row">
-  <div class="col-md-12">
-    <div class="panel panel-fixity panel-classify-work">
-      <div class="panel-heading">Fixity</div>
-      <div class="panel-body">
-        <ul>
-          <li><%= link_to 'Fixity Dashboard', fixity_dashboard_path %></li>
-        </ul>
+<% if current_or_guest_user.admin? %>
```
Comment (tpendragon) [inline app/views/catalog/_fixity_dashboard_home.html.erb]: Can you just do `<% if can?(:read, :fixity) %>`?

Comment (tpendragon) [reply app/views/catalog/_fixity_dashboard_home.html.erb]: Also can we get a spec for viewing the link?

Comment (hackmastera) [reply app/views/catalog/_fixity_dashboard_home.html.erb]: oh yeah totally. I put that in the controller after restricting the view. will change it to use `can?`

Comment (hackmastera) [reply app/views/catalog/_fixity_dashboard_home.html.erb]: spec added. ready for re-review.

**Comment by tpendragon** [comment]
@hackmastera You said you had failing feature specs, but CI looks happy - I'll let you hit the button if it's a local issue.

**Comment by hackmastera** [comment]
my problem was due to mismatch in versions between chrome browser and chromedriver. this fixed it https://github.com/flavorjones/chromedriver-helper#updating-to-latest-chromedriver

**Pull request merged**